test/libsolidity/SolidityEndToEndTest.cpp

```diff
@@ -1650,37 +1606,38 @@ BOOST_AUTO_TEST_CASE(bytes_from_calldata_to_memory)
 	);
 }
 
-BOOST_AUTO_TEST_CASE(call_forward_bytes)
-{
-	char const* sourceCode = R"(
-		contract receiver {
-			uint public received;
-			function recv(uint x) public { received += x + 1; }
-			fallback() external { received = 0x80; }
-		}
-		contract sender {
-			constructor() { rec = new receiver(); }
-			fallback() external { savedData = msg.data; }
-			function forward() public returns (bool) { address(rec).call(savedData); return true; }
-			function clear() public returns (bool) { delete savedData; return true; }
-			function val() public returns (uint) { return rec.received(); }
-			receiver rec;
-			bytes savedData;
-		}
-	)";
-	ALSO_VIA_YUL(
-		DISABLE_EWASM_TESTRUN();
-		compileAndRun(sourceCode, 0, "sender");
-		ABI_CHECK(callContractFunction("recv(uint256)", 7), bytes());
-		ABI_CHECK(callContractFunction("val()"), encodeArgs(0));
-		ABI_CHECK(callContractFunction("forward()"), encodeArgs(true));
-		ABI_CHECK(callContractFunction("val()"), encodeArgs(8));
-		ABI_CHECK(callContractFunction("clear()"), encodeArgs(true));
-		ABI_CHECK(callContractFunction("val()"), encodeArgs(8));
-		ABI_CHECK(callContractFunction("forward()"), encodeArgs(true));
-		ABI_CHECK(callContractFunction("val()"), encodeArgs(0x80));
-	);
-}
+// // extractable
+// BOOST_AUTO_TEST_CASE(call_forward_bytes)
```

Comment:
Why is this commented out?

Reply:
Updates : 
1. Removed the libevmone.so* files. 
2. The commented regions are only there for this draft PR. I was looking to use this as a safety net in case I needed to revert any particular test case made, since I extracted 7 test cases in the first commit itself. An alternate to this should have been to extract one test case per commit. I'll be following that from now on.

Reply:
If you want to revert a part of the commit you can always to an interactive rebase, set the commit to `edit` and amend the commit.